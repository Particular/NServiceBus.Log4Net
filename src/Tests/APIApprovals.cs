using NServiceBus;
using NUnit.Framework;
using Particular.Approvals;
using PublicApiGenerator;

[TestFixture]
public class APIApprovals
{
    [Test]
    public void Approve()
    {
#pragma warning disable 0618
        var publicApi = ApiGenerator.GeneratePublicApi(typeof(Log4NetFactory).Assembly, excludeAttributes: new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" });
#pragma warning restore 0618
        Approver.Verify(publicApi);
    }
}