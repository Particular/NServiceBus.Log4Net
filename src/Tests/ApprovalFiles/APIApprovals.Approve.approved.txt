namespace NServiceBus
{
    public class Log4NetFactory : NServiceBus.Logging.LoggingFactoryDefinition
    {
        public Log4NetFactory() { }
        protected override NServiceBus.Logging.ILoggerFactory GetLoggingFactory() { }
    }
}